packages/storefront-lwc/src/modules/commerce/productDetail/productDetail.js

```diff
@@ -30,7 +31,11 @@ export default class ProductDetail extends LightningElement {
         selectedColor: '',
     };
 
-    @api set pid(val) {
```

Comment:
why don't we need the @api any more for this setter?

Reply:
Because the `pid` value isn't getting passed directly to the component anymore, rather it is getting wired in through an adapter